docs/cookbook/osm/index.rst

```diff
@@ -0,0 +1,71 @@
+OpenStreetMap
+=============
+
+Problem
+-------
+
+You'd like to use an alternative base map, such as OpenStreetMap, for cartographic styling reasons or to provide a suitable base map on an isolated network.
+
+
+Solution
+--------
+
+Modify the config settings to specify an additional or alternative base map provider.
+
+The following code shows how to change the config settings to add an additional basemap. Note that the existing "streetmap" and "worldimagery" parts, and other elements, are skipped to simplify the presentation. You can make other changes to meet your requirements, but removing existing items is not necessary to add a new provider.
+
+.. code-block:: json
+  :caption: ``config/settings.json``
+
+  {
+    "providers": {
+      "basemap": {
+        "maps": {
+          "osm": {
+            "title": "OpenStreetMap",
+            "type": "BaseMap",
+            "baseType": "XYZ",
+            "provider": "OSM",
+            "url": "//c.tile.openstreetmap.org/{z}/{x}/{y}.png",
```

Comment:
Does OSM have `//[a-c].tile.openstreetmap.org/`? If so, we should use more than just `c`.

I think single URL templating is currently broken in OpenSphere (and we should write an issue for that), but you can do:

```
"urls": [
  "//a.tile.openstreetmap.org/{z}/{x}/{y}.png",
  "//b.tile.openstreetmap.org/{z}/{x}/{y}.png",
  "//c.tile.openstreetmap.org/{z}/{x}/{y}.png"
]
```

Reply:
Yeah, it does. I'll test templating (didn't see any code to support it), and raise the ticket as appropriate.

Will update the PR to include multiple URLs.

Reply:
Actually, it looks like URL array is also a little tricky (#86). Will see if I can fix that up.

Reply:
Urgh. `urls`, not `url`....